src/utils/timeRangeUtils.ts

```diff
@@ -64,9 +65,18 @@ const getDefaultTimeRangeOption = (
 	return selectedTimeRange ? selectedTimeRange : defaultTimeRangeOption;
 };
 
+//accepts a date-time string and outputs a human readable string with timezone
+//output format 31/12/1990 11:59 pm IST
+const getDateTimeWithTZ = (dateTime: string, format: string = 'DD/MM/YYYY h:mm a z') => {
+	const localTimeZone = moment.tz.guess();
+	const convertedDate = moment.tz(dateTime, localTimeZone);
+
+	return convertedDate.format(format);
```

Comment:
Now the errors are silenced. 
Since the dateTime arg is a string and not Date obj itself, It still can return 'Invalid date'. 
So check if its valid and return the formatted date or null.